src/tests/createHeadlessForm.test.js

```diff
@@ -1578,6 +1579,44 @@ describe('createHeadlessForm', () => {
       });
     });
 
+    it('supports "null" field type', () => {
+      const { handleValidation, fields } = createHeadlessForm(schemaInputTypeNull, {
+        strictInputType: false,
+      });
+
+      expect(fields).toMatchObject([
+        {
+          name: 'name',
+          label: '(Optional) Name',
+          type: undefined,
+          jsonType: 'null',
```

Comment:
Let's assert here that `inputType` is undefined based on your last discovery ^.^

Reply:
In case you are wondering, `type` is tech-debt/deprecated. It should not exist (but we still use it a lot in our internal codebases). It's either `inputType` or `jsonType`.

Reply:
Good idea! Done!